### PR TITLE
 Document multi-issuer contracts according to NatSpec guidelines

### PR DESCRIPTION
**Pull request by walkerq**

### CODE REVIEW

File: contracts/minting/MasterMinter.sol [hunk 1]
```diff
@@ -24,6 +24,10 @@ pragma solidity ^0.4.24;
 
 import "./MintController.sol";
 
+/**
+ * @title MasterMinter
+ * @dev Default implementation of the MintController contract.
```
Comment (mirathewhite) [inline contracts/minting/MasterMinter.sol]: @notice MasterMinter uses multiple controllers to manage minters for a contract that implements the MinterManagerInterface.
@dev MasterMinter inherits all its functionality from MintController.

File: contracts/minting/MintController.sol [hunk 2]
```diff
@@ -28,12 +28,17 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 /**
  * @title MintController
- * @dev allows control of configure/remove minter by different addresses
- *
+ * @dev Implementation of the abstract Controller contract, in which
```
Comment (mirathewhite) [inline contracts/minting/MintController.sol]: @notice The MintController contract manages minters for a contract that implements the MinterManagerInterface.  It lets the owner designate certain addresses as controllers, and these controllers then manage the minters by adding and removing minters, as well as modifying their minting allowance.  A controller may manage exactly one minter, but the same minter address may be managed by multiple controllers.
@dev MintController inherits from the Controller contract.  It treats the Controller workers as minters.

File: contracts/minting/MinterManagementInterface.sol [hunk 3]
```diff
@@ -5,27 +5,37 @@
...
+* THE SOFTWARE.
 */
 
 pragma solidity ^0.4.24;
 
-// Using an interface for managing minters so that MintController
-// can be used for managing minters with different contracts.
+/** 
+* @title MinterManagementInterface
+* @dev Interface for managing the allowances of minters.
```
Comment (mirathewhite) [inline contracts/minting/MinterManagementInterface.sol]: @notice A contract the implements the MinterManagementInterface has external functions for adding and removing minters and modifying their allowances.  An example of such a contract is the FiatTokenV1 contract that
implements USDC.

**Comment by walkerq** [comment]
@mirathewhite thanks for the suggestions - I added them all!

**Comment by walkerq** [comment]
@mirathewhite Thanks for the suggestions! I added all of them.

File: contracts/minting/Controller.sol [hunk 4]
```diff
@@ -27,11 +27,14 @@ import "../Ownable.sol";
 /**
  * @title Controller
  * @dev Generic implementation of the owner-controller-worker model.
- *
+ * One owner manages many controllers. Each controller manages one worker.
```
Comment (mirathewhite) [inline contracts/minting/Controller.sol]: replace @dev with @notice

File: contracts/minting/Controller.sol [hunk 5]
```diff
@@ -27,11 +27,14 @@ import "../Ownable.sol";
...
+ * Workers may be reused across different controllers.
  */
 contract Controller is Ownable {
-    // controllers[controller] = worker
-    // The controller manages a single worker address.
+    /**
+    * @dev A controller manages a single worker address.
```
Comment (mirathewhite) [inline contracts/minting/Controller.sol]: replace @dev with @notice

File: contracts/minting/Controller.sol [hunk 6]
```diff
@@ -41,7 +44,7 @@ contract Controller is Ownable {
     event ControllerRemoved(address indexed _controller);
 
     /**
-     * @dev ensure that caller is the controller of a non-zero worker address
+     * @dev ensures that caller is the controller of a non-zero worker address.
```
Comment (mirathewhite) [inline contracts/minting/Controller.sol]: replace @dev with @notice

File: contracts/minting/MintController.sol [hunk 7]
```diff
@@ -140,6 +157,7 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
+    // Internal functions
     /**
      * @dev decreases the minter allowance if and only if the minter is
```
Comment (mirathewhite) [inline contracts/minting/MintController.sol]: replace @dev with @notice

File: contracts/minting/MintController.sol [hunk 8]
```diff
@@ -171,11 +189,11 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
-   // Internal functions
-
     /**
      * @dev Uses the MinterManagementInterface to enable the minter and
```
Comment (mirathewhite) [inline contracts/minting/MintController.sol]: replace @dev with @notice

Comment (walkerq) [reply contracts/minting/MintController.sol]: Updated with your latest round of comments @mirathewhite

File: contracts/minting/Controller.sol (hunk 5 above)
Comment (walkerq) [reply contracts/minting/Controller.sol]: Updated

File: contracts/minting/Controller.sol (hunk 6 above)
Comment (walkerq) [reply contracts/minting/Controller.sol]: Updated

File: contracts/minting/MintController.sol (hunk 7 above)
Comment (walkerq) [reply contracts/minting/MintController.sol]: Updated

File: contracts/minting/MintController.sol (hunk 8 above)
Comment (walkerq) [reply contracts/minting/MintController.sol]: Updated

**Pull request merged**